modules/rosa-installing-prerequisites.adoc

```diff
@@ -25,12 +25,12 @@ This user will have Programmatic access enabled and the AdministratorAccess poli
 +
 . Install and configure the AWS CLI.
 .. Follow the link:https://aws.amazon.com/cli/[AWS Command Line Interface] documentation to install and configure the AWS CLI for your operating system.
-.. Optional: Set a default AWS region. `rosa` evaluates regions in the following priority order:
+.. (Optional) Set a default AWS region. `rosa` evaluates regions in the following priority order:
```

Comment:
Another weird one for our recent style decisions is going with:

. Optional: <step>